### PR TITLE
[Notifications] feat: implement funds claimed noticiations

### LINKED ISSUE #3190: Notifications: Incoming funds claimed notification
### Story

Example: As a *user with notifications enabled*, I want to *see a notification when incoming funds are claimed in my colonies*, so that *I am kept informed about the balance of my colonies updating*

### Description

When incoming funds are claimed by a colony, a notification should be sent to any users in the colony who have notifications enabled, and this colony not muted.

### This issue is complete when...

- [ ] A notification is sent to colony users when incoming funds are claimed

### PR DESCRIPTION
**Pull request by bassgeta**
## Description

This PR implements notifications when colony funds are claimed. Initially we wanted to discern if the funds were minted, but it's not that easy since we always claim all funds, and you can claim multiple incoming funds by a single minting action, so we skipped that.

[Block ingestor PR](https://github.com/JoinColony/block-ingestor/pull/274)

## Testing

Basically let's try out two flows, one for minting tokens, the other one for manually claiming incoming funds
1. Go to the incoming funds page and claim incoming GG funds
image
2. After a short while you should get an indicator and a new notification :eyes: 
image
image
3. Clicking on it should take you to the balances page and mark it as read
image
4. Now go ahead and mint 13 million CREDS so we check the formatting too
image
5. After a short while the notification should arrive
image

## Diffs

**New stuff** ✨

* `FundsClaimedNotification` component

**Changes** 🏗

* moved the `getNumeralTokenAmount` helper to token utils (was previously a util function in `CompletedAction`

Resolves #3190

### CODE REVIEW

File: src/components/common/Extensions/UserHub/partials/NotificationsTab/partials/Notification/FundsClaimed/FundsClaimedNotificationMessage.tsx
Comment (davecreaser) [inline src/components/common/Extensions/UserHub/partials/NotificationsTab/partials/Notification/FundsClaimed/FundsClaimedNotificationMessage.tsx:1]: 🌮

**Review by davecreaser — APPROVED** [review]
Perfect! Just one minor comment and the empty file to delete.

Screenshots to prove it's working:
Screenshot 2024-10-08 at 17 19 51
Screenshot 2024-10-08 at 17 21 09

And one note: I know it's confusing at the moment because mint tokens takes the fee into account in the messaging whereas the incoming funds claim doesn't. I've fixed that in my next PR so it'll be all good :)

Great job! Thanks for picking up yet another notification ticket 📈

**Review by rdig — APPROVED** [review]
Tested and everything works as expected. Nicely done! :100: 

Screenshot from 2024-10-09 11-54-00
Screenshot from 2024-10-09 11-54-14
Screenshot from 2024-10-09 11-54-29
Screenshot from 2024-10-09 11-55-03
Screenshot from 2024-10-09 11-55-11

**Review by iamsamgibbs — APPROVED** [review]
Nice job, this works great! Appreciate all the code tidy up too!

Screenshot 2024-10-10 at 14 03 56

Navigates to the balances page:

https://github.com/user-attachments/assets/cbed24b5-3049-49c1-bed4-e6b1fec362fd

Works when minting:

Screenshot 2024-10-10 at 14 05 18

Might want to consider adding the `CloseUserHub` function as I have in #3290 but we can just sneak this in at a later point (or I should remove it to unify this behaviour, but it feels more natural to me that it closes when navigating to another page, and it is already closed when opening a transaction).

**Pull request merged**